db/migrate/20160225114042_add_apointment_type_to_appointment_summaries.rb

```diff
@@ -0,0 +1,5 @@
+class AddApointmentTypeToAppointmentSummaries < ActiveRecord::Migration
```

Comment:
We need to migrate the database before the code that depends on it goes into production. To do that we need to isolate the migration and schema changes into a separate PR. We can then make those changes in production.